### PR TITLE
HTMD tutorial fixes prior to BCC

### PR DESCRIPTION
**Pull request by simonbray**

### CODE REVIEW

File: topics/computational-chemistry/tutorials/htmd-analysis/tutorial.md [hunk 1]
```diff
@@ -148,9 +148,16 @@ Please note all GROMACS tools output a log. Generally, you only need to look at
...
+
 > ### {% icon hands_on %} Hands-on: Generate ligand topology
 >
-> 1. **Generate MD topologies for small molecules** {% icon tool %} with the following parameters:
+> 1. **Compound conversion** {% icon tool %} with the following parameters:
+>    - *"Molecular input file"*: 'Ligand (PDB)'
+>    - *"Output format"*: `Protein Data Bank format (pdb)`
+>    - *"Add hydrogens appropriate for pH"*: `7.0`
+> 2. Rename the output file to `Hydrated ligand (PDB)`.
+> 2. **Generate MD topologies for small molecules** {% icon tool %} with the following parameters:
```
Comment (chrisbarnettster) [inline topics/computational-chemistry/tutorials/htmd-analysis/tutorial.md:160]: Check numbering here . 2. and 2. ?

**Comment by simonbray** [comment]
Thanks @chrisbarnettster. No need to merge yet, I will continue to add to this PR.

**Comment by simonbray** [comment]
Okay, ready to be merged.

@chrisbarnettster @tsenapathi any more comments / changes?

**Comment by simonbray** [comment]
Thanks @tsenapathi!

**Pull request merged**